packages/core/src/browser/theming.ts

```diff
@@ -93,7 +93,7 @@ export class ThemeService {
     }
 
     getTheme(themeId: string): Theme {
-        return this.themes[themeId] || this.defaultTheme;
+        return this.themes[themeId] || undefined;
```

Comment:
What about the return type: `Theme | undefined`?

Reply:
updated

Reply:
I will expect a build failure. You haven't updated the call sites.

Also, if I am not mistaken, there is absolutely no need to do `this.themes[themeId] || undefined;`. `this.themes[themeId]` is sufficient, it will be `undefined` anyways.

Reply:
@kittaakos 
Can you check it again? I updated the code accordingly.